app/controllers/application_controller.rb

```diff
@@ -16,6 +14,7 @@ def disable_fine_print
   end
 
   def check_if_admin
+    return true if !Rails.env.production?
```

Comment:
Does this allow anyone to be admin? I see there is a check for Blazer using this, but no where else.

Reply:
yes but only in non-prod environments (so only locally - since everything else runs as ENV=production)

Reply:
and yes, it's only used for Blazer auth checking